community/neo4j-harness/src/main/java/org/neo4j/harness/internal/InProcessServerBuilder.java

```diff
@@ -19,48 +19,17 @@
  */
 package org.neo4j.harness.internal;
 
-import org.apache.commons.io.FileUtils;
-
 import java.io.File;
-import java.io.IOException;
-import java.io.OutputStream;
-import java.util.HashMap;
 import java.util.Map;
-import java.util.Random;
 
-import org.neo4j.function.Function;
-import org.neo4j.graphdb.GraphDatabaseService;
-import org.neo4j.graphdb.config.Setting;
-import org.neo4j.graphdb.factory.GraphDatabaseSettings;
-import org.neo4j.harness.ServerControls;
-import org.neo4j.harness.TestServerBuilder;
-import org.neo4j.io.fs.DefaultFileSystemAbstraction;
-import org.neo4j.io.fs.FileSystemAbstraction;
-import org.neo4j.kernel.GraphDatabaseDependencies;
 import org.neo4j.kernel.configuration.Config;
 import org.neo4j.kernel.impl.factory.GraphDatabaseFacadeFactory;
 import org.neo4j.logging.FormattedLogProvider;
+import org.neo4j.server.AbstractNeoServer;
 import org.neo4j.server.CommunityNeoServer;
-import org.neo4j.server.configuration.ServerSettings;
-
-import static org.neo4j.io.file.Files.createOrOpenAsOuputStream;
-import static org.neo4j.server.configuration.ConfigurationBuilder.ConfiguratorWrappingConfigurationBuilder.toStringForThirdPartyPackageProperty;
-import static org.neo4j.server.configuration.Configurator.DATABASE_LOCATION_PROPERTY_KEY;
-import static org.neo4j.server.configuration.Configurator.WEBSERVER_PORT_PROPERTY_KEY;
-import static org.neo4j.test.Digests.md5Hex;
 
-public class InProcessServerBuilder implements TestServerBuilder
+public class InProcessServerBuilder extends AbstractInProcessServerBuilder
```

Comment:
This will break backawrd-compatibility if there are anyone who is extending `InProcessServerBuilder`. If this is OK, then I will rename this class.